src/Resources/views/Profiler/block.html.twig

```diff
@@ -112,21 +109,6 @@
                 <td>{{ block.duration|number_format(2) }} ms</td>
             </tr>
 
-            {% if block.cache.handler %}
```

Comment:
@VincentLanglet so this is never true without the caching functionality. 

I don't think this is meaningful anymore. So I propose to cleanup the trace data and remove the caching related data.

Reply:
Ok